geth-utils/l2geth/trace.go

```diff
@@ -186,6 +198,7 @@ func Trace(config TraceConfig) (*types.BlockTrace, error) {
 	// fmt.Printf("geth-utils: ArchimedesBlock = %d\n", chainConfig.ArchimedesBlock)
 
 	txs := transferTxs(config.Transactions, chainConfig.ChainID)
+	fmt.Printf("geth-utils: transferTxs = %v\n", txs)
```

Comment:
really needed?  I noticed the l2geth module does not print other info?

Reply:
previously for debugging  removed [93967d5](https://github.com/scroll-tech/zkevm-circuits/pull/1412/commits/93967d5fec533fbf2a1b573b87b7dcca4fa38f58)